src/DotNet/ReflectionExtensions.cs

```diff
@@ -105,5 +106,33 @@ static class ReflectionExtensions {
 		/// <param name="type">this</param>
 		public static bool IsTypeDef(this Type type) =>
 			!(type is null) && !type.HasElementType && (!type.IsGenericType || type.IsGenericTypeDefinition);
+
+		internal static string Unescape(string name) {
+			if (string.IsNullOrEmpty(name) || !name.Contains(@"\"))
```

Comment:
`name.IndexOf('\\') < 0` is better since it just searches for one char instead of a substring.